src/test/java/io/openliberty/tools/intellij/it/SingleModMPProjectCfgTestCommon.java

```diff
@@ -0,0 +1,189 @@
+/*******************************************************************************
+ * Copyright (c) 2025 IBM Corporation.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.tools.intellij.it;
+
+import com.automation.remarks.junit5.Video;
+import com.intellij.remoterobot.RemoteRobot;
+import org.junit.jupiter.api.*;
+
+import java.io.File;
+import java.nio.file.Paths;
+import java.time.Duration;
+
+import static com.intellij.remoterobot.utils.RepeatUtilsKt.waitForIgnoringError;
+
+/**
+ * Holds common tests that use a single module MicroProfile project.
+ */
+public abstract class SingleModMPProjectCfgTestCommon {
+
+    // In this test case the environment has been set up so that there is a new project
+    // that has not been used in a previous execution of IntelliJ. Also, the Liberty explorer
+    // or dashboard has not been opened as it is in all other tests. This means the LibertyModules
+    // object is not yet populated.
+    // When we create a new Run/Debug configuration, the "Liberty project" field is populated by
+    // default with one of the build files from one of the Liberty projects in the workspace. If
+    // there is no default build file then there will be a Null Pointer Exception if we press Run.
+    // The following method will throw a NoSuchElementException if there are no Liberty projects
```

Comment:
which method is this sentence referring to?

Reply:
I moved the comment to the location we call the method. It doesn't need to be at the top like that.